informer/collector.go

```diff
@@ -0,0 +1,65 @@
+package informer
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"k8s.io/apimachinery/pkg/api/meta"
+)
+
+var (
+	description *prometheus.Desc = prometheus.NewDesc(
+		prometheus.BuildFQName("operatorkit", "informer", "deletion_timestamp"),
+		"DeletionTimestamp of watched objects.",
+		[]string{
+			"name",
+			"namespace",
+		},
+		nil,
+	)
+)
+
+// Describe is used to describe metrics which are exported to Prometheus.
+func (i *Informer) Describe(ch chan<- *prometheus.Desc) {
+	ch <- description
+}
+
+// Collect is called by the Prometheus registry when collecting metrics from
+// the informer.
+func (i *Informer) Collect(ch chan<- prometheus.Metric) {
+	i.logger.Log("level", "debug", "message", "start collecting metrics")
```

Comment:
The equivalent for the end of the action is also missing here.

Reply:
Already exists:
https://github.com/giantswarm/operatorkit/blob/60eae9c770b959337a5aa4c29eaf15e387a8910e/informer/collector.go#L61

Reply:
Ah yeah, I see. Did not notice while scanning because it is "hidden" in the loop/select. All good.